modules/ROOT/pages/prereq-hardware.adoc

```diff
@@ -182,7 +182,7 @@ In each case, your load balancer must listen on the load balancer port and redir
 Your load balancer should poll the address `HTTP:10248/healthz` to run a health check on your platform servers and confirm that they are accessible.
 
 [NOTE]
-Do not configure SSL certificates in your load balancer. TLS termination is handled by the platform with the certificates configured using Access Management. See xref:config-workflow.adoc[Post-installation Tasks].
+Do not configure SSL certificates in your load balancer. TLS termination is handled by the platform with the certificates configured using Access Management. See xref:config-workflow.adoc[Configure Anypoint Platform PCE].
```

Comment:
Try to use a more precise term instead of "handle," per style ref.:
```suggestion
Do not configure SSL certificates in your load balancer. TLS termination is performed by the platform, with the certificates configured using Access Management. See xref:config-workflow.adoc[Configure Anypoint Platform PCE].
```
OR, to avoid passive voice:
```suggestion
Do not configure SSL certificates in your load balancer. The platform performs TLS termination by configuring certificates using Access Management. See xref:config-workflow.adoc[Configure Anypoint Platform PCE].
```